src/main/webapp/components/utils/getField.js

```diff
@@ -0,0 +1,9 @@
+/**
+  * Returns the value at obj[field] or defaultValue if obj[field] === undefined.
+  * Useful for extracting a field from an object, with a fallback.
+  *
+  * @param obj the object to query the fields for.
+  * @param field the target field, in string form. 
+  * @param defaultValue the value to be used if obj[field] === undefined.
+ */
+export const getField = (obj, field, defaultValue) => obj[field] !== undefined ? obj[field] : (defaultValue !== undefined ? defaultValue : null);
```

Comment:
please have the [column limit to 80](https://google.github.io/styleguide/jsguide.html#formatting-column-limit). Applies here and to the other files.

Reply:
Columns truncated 68661b1, d044224, 247b20a and 9e8fd81